docker/linux/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM philcryer/min-jessie:latest
```

Comment:
With my manjaro (arch) I had to resort to compile like this (without using docker scripts from you). As it turns out the current build doesn't support yet latest gcc (7)